### DISCUSSION TITLE
What files are needed to make it look example

### DISCUSSION THREAD

**Discussion by cadop** [jothepro/doxygen-awesome-css#77] [General]
I need an ELI5 for this.  

I tried following the instructions which give two options, one to clone the entire repo, and one to just copy a single .css file. I did the latter, but it doesn't work. I assume there is more that also needs to be included. However, I don't want to include readme and another doxyfile by cloning this repo.  So  is there a list of files and organization I need to follow to get the same result?

Thanks!

**Comment by cadop** [comment]
So, here it is, for others that were as confused as me:

Yes, you only need 1 file. This is [doxygen-awesome.css](https://github.com/jothepro/doxygen-awesome-css/blob/main/doxygen-awesome.css).  You put it in a folder that doxygen can access, following the example.  

Important: If you are trying to use the sidebar-only version, its not an either/or. You need *both* the `doxygen-awesome.css` *and* `doxygen-awesome-sidebar-only.css`.   It does show this in the readme, but obviously my brain skipped that both are included. 

Theme looks great!

**Reply by jothepro** [reply]
Nice, cool that you could make it work! If you have suggestions on how the documentation or code structure could be improved in these regards to avoid confusion, pls let me know of open a PR! As the author of the code one can get kind of blind in these regards. 😉

**Reply by cadop** [reply]
Sure I opened a PR with what I think would have prevented my confusion
